gtfs-realtime/spec/en/reference.md

```diff
@@ -140,7 +140,7 @@ Uncertainty applies equally to both time and delay. The uncertainty roughly spec
 Realtime update for arrival and/or departure events for a given stop on a trip. Please also refer to the general discussion of stop time updates in the [TripDescriptor](#message-tripdescriptor) and [trip updates entities](trip-updates.md) documentation.
 
 Updates can be supplied for both past and future events. The producer is allowed, although not required, to drop past events.
-The update is linked to a specific stop either through stop_sequence or stop_id, so one of these fields must necessarily be set.
+The update is linked to a specific stop either through stop_sequence or stop_id, so one of these fields must necessarily be set.  If the same stop_id is visited more than once in a trip, then stop_sequence must be provided in all StopTimeUpdates for that stop_id on that trip.
```

Comment:
>Should there be a similar update to trip-updates.md line 18?

Good catch!  Just forced pushed a new commit that includes this addition to trip-updates.md as well.

>I might degrade 'must' to 'should' though.

In the commit I just force-pushed I also changed "must" to "should".